src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+/**
+ * This class deals with user's input by parsing them and convert each instruction
+ * into different commands.
+ */
+public class Parser {
+    private enum KeyWord {
+        END("bye"),
+        DELETE("delete"),
+        LIST("list"),
+        GET("get"),
+        SEARCH("find"),
+        DONE("done");
+
+        private final String k;
```

Comment:
Since `String k` is an attribute of `KeyWord`, maybe we need to make its name more precise?